bookkeeper-server/src/test/java/org/apache/bookkeeper/test/BookKeeperClusterTestCase.java

```diff
@@ -833,4 +834,13 @@ public TestStatsProvider getStatsProvider(int index) throws Exception {
         return getStatsProvider(bs.get(index).getLocalAddress());
     }
 
+    public static int countNumOfFiles(File[] folderNames, String... extensions) {
```

Comment:
move this utility method to bookkeeper/bookkeeper-server/src/test/java/org/apache/bookkeeper/util/TestUtils.java